platform/ui/src/types/index.ts

```diff
@@ -15,3 +14,9 @@ const StringNumber = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
 const StringArray = PropTypes.oneOfType([PropTypes.string, PropTypes.array]);
 
 export { StringNumber, StringArray, ThumbnailType };
+
+export type IProps = Record<string, unknown>;
+
+export type Predicate = (props: IProps) => boolean;
+
+export type Object = Record<string, unknown>;
```

Comment:
I don't think these add any useful type saftey other than just falsly giving the impression of being type safe. What is wrong with `Record<string, unknown>` ?